src/index.d.ts

```diff
@@ -544,6 +544,48 @@ declare namespace admin.auth {
     toJSON(): Object;
   }
 
+  /**
+   * Interface representing the common properties of a user enrolled second factor.
+   */
+  interface MultiFactorInfo {
+  
+    /**
+     * The user's unique ID of the enrolled second factor.
+     */
+    uid: string;
+
+    /**
+     * The optional display name for an enrolled second factor.
+     */
+    displayName?: string;
+
+    /**
+     * The optional date the second factor was enrolled, formatted as a UTC string.
+     */
+    enrollmentTime?: string;
+
+    /**
+     * The second factor type identifier. For SMS second factor, this is `"phone"`.
```

Comment:
The type identifier of the second factor. For SMS second factors, this is `phone`.

Reply:
Done.